src/_utils/formValidation.js

```diff
@@ -0,0 +1,6 @@
+const emailValidation = (value) => {
+  const emailRegex = /^\w+([.-]?\w+)*@\w+([.-]? w+)*(\.\w{2,3})+$/;
+  return !!emailRegex.test(value);
```

Comment:
Why the double `!!` ?

Reply:
There's no need for the `!!`.  `test` returns true or false. Check the explanation in [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test)

Reply:
Okay.